packages/lesswrong/components/sunshineDashboard/SunshineNewUsersItem.jsx

```diff
@@ -15,6 +15,10 @@ import { withStyles } from '@material-ui/core/styles';
 const styles = theme => ({
   negativeKarma: {
      color: red['A100']
+  },
+  info: {
+    wordBreak: "break-word",
```

Comment:
Seems to work in Chrome, but caveat: `break-word` is nonstandard and [isn't supported](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) in Firefox. Given that this is a moderator-only part of the site, and things don't break too bad if this is left out, that's probably fine; but it's a good dev-habit to check MDN whenever you use obscure CSS.

```suggestion
    // Wrap between MetaInfo elements. Non-standard CSS which may not work in Firefox.
    wordBreak: "break-word",
```